content/docs/command-reference/exp/list.md

```diff
@@ -10,7 +10,7 @@ usage: dvc exp list [-h] [-q | -v] [--rev <rev>]
                     [git_remote]
 
 positional arguments:
-  git_remote    Optional Git remote name or repo URL
+  git_remote    Optional Git remote name or valid Git URL
```

Comment:
```suggestion
  git_remote    Optional Git remote name or Git URL (SSH auth only)
```

Reply:
But what about: iterative/dvc#6493 (comment) . This change would only really apply to push, right?

Reply:
@daavoo As your user guide comment explains, this applies to reads also (for example, private repos), although authentication may not be necessary for reads as often as for writes. Do you think the suggested change is unclear?

Reply:
I think we can say "authenticated Git URL" without mentioning SSH. The requirement is _authentication_, not a particular protocol. It may be a `file://...` URL on NFS and it still works if you're authenticated, and doesn't if you're not. The reason we can't use HTTP is that we don't get user credentials for authentication and HTTP doesn't allow unauthenticated writes. I believe it's better to mention "authentication"  rather than SSH here.

Reply:
@dberenbaum  "SSH Git URL" was my first proposed change and then @pmrowla clarified that, for reading (`list`, `pull`), "SSH" is not a requirement in some cases. So I'm confused about getting back to that terminology. 

@iesahin I like "authenticated" very much 👍

Reply:
> The requirement is _authentication_, not a particular protocol.

👍  I'm not sure "authenticated" sounds right. I would interpret that as meaning that DVC won't work with URLs where authentication is not required, like reading from a public Github url, or writing to an unprotected NFS URL.

What needs to be clear is that DVC will not work if credentials need to be provided to authenticate (see https://git-scm.com/docs/gitcredentials). Unfortunately, even this may be confusing because we aren't currently using Git's credentials helpers, so even if someone has their credentials stored in Git and doesn't need to enter credentials when doing `git push`, `dvc exp push` may still fail. I opened https://github.com/iterative/dvc/issues/6586 to possibly implement support for credentials, which would negate this whole PR and discussion 😄 .

For now, I think it's fine to do any of:

1. Close this PR altogether since https://github.com/iterative/dvc/pull/6493 should make the problem more self-explanatory.
2. Add a note that the command will fail if credentials need to be provided to authenticate.
3. Merge as is if I'm the only one who finds "authenticated" confusing.

Reply:
> The requirement is authentication

Authentication is NOT always a requirement from what I understand? So "authenticated Git URL" (current text) seems equally misleading... I agree with the statement in https://github.com/iterative/dvc/pull/6493#issuecomment-914431340:

> we could probably summarize behavior as not supporting HTTP authentication

So maybe for now we can go with `Git URL (HTTP auth is not supported)`

Reply:
If that's not convincing let's just go with something simple like `Git URL` and let the error happen in the edge cases, and explain the details in the docs. Hard to be super correct in a short description.

Reply:
Changed the following:

- Just using `Git URL` in command reference.
- Added note about git credentials in the troubleshooting section.

And let users reach the troubleshooting page when the exception is raised